src/views/Entities/InternalServices.vue

```diff
@@ -0,0 +1,383 @@
+            :has-error="entityHasError"
+            :is-loading="entityIsLoading"
+            :is-empty="entityIsEmpty"
+          >
+            <div>
+              <ul>
+                <li
+                  v-for="(val, key) in entity"
+                  :key="key"
+                >
+                  <h4>{{ key }}</h4>
+                  <p>
+                    {{ val }}
+                  </p>
+                </li>
+              </ul>
+            </div>
+          </LabelList>
+        </template>
+        <template slot="yaml">
+          <YamlView
+            lang="yaml"
+            :title="entityOverviewTitle"
+            :has-error="entityHasError"
+            :is-loading="entityIsLoading"
+            :is-empty="entityIsEmpty"
+            :content="rawEntity"
+          />
+        </template>
+      </Tabs>
+    </FrameSkeleton>
+  </div>
+</template>
+
+<script>
+import { getOffset, getSome, stripTimes } from '@/helpers'
+import EntityURLControl from '@/components/Utils/EntityURLControl'
+import sortEntities from '@/mixins/EntitySorter'
+import FormatForCLI from '@/mixins/FormatForCLI'
+import FrameSkeleton from '@/components/Skeletons/FrameSkeleton'
+import Pagination from '@/components/Pagination'
+import DataOverview from '@/components/Skeletons/DataOverview'
+import Tabs from '@/components/Utils/Tabs'
+import YamlView from '@/components/Skeletons/YamlView'
+import LabelList from '@/components/Utils/LabelList'
+
+export default {
+  name: 'InternalServices',
+  metaInfo: {
+    title: 'Internal Services'
+  },
+  components: {
+    EntityURLControl,
+    FrameSkeleton,
+    Pagination,
+    DataOverview,
+    Tabs,
+    YamlView,
+    LabelList
+  },
+  mixins: [
+    FormatForCLI,
+    sortEntities
+  ],
+  data () {
+    return {
+      isLoading: true,
+      isEmpty: false,
+      hasError: false,
+      entityIsLoading: true,
+      entityIsEmpty: false,
+      entityHasError: false,
+      tableDataIsEmpty: false,
+      empty_state: {
+        title: 'No Data',
+        message: 'There are no Internal Services present.'
+      },
+      tableData: {
+        headers: [
+          { key: 'actions', hideLabel: true },
+          { label: 'Name', key: 'name' },
+          { label: 'Mesh', key: 'mesh' },
+          { label: 'Data plane proxies: Online / Total', key: 'totalOnline' },
+        ],
+        data: []
+      },
+      tabs: [
+        {
+          hash: '#overview',
+          title: 'Overview'
+        },
+        {
+          hash: '#yaml',
+          title: 'YAML'
+        }
+      ],
+      entity: [],
+      rawEntity: null,
+      firstEntity: null,
+      pageSize: this.$pageSize,
+      pageOffset: null,
+      next: null,
+      hasNext: false,
+      previous: []
+    }
+  },
+  computed: {
+    tabGroupTitle () {
+      const entity = this.entity
+
+      if (entity) {
+        return `Internal Services: ${entity.name}`
+      } else {
+        return null
+      }
+    },
```

Comment:
I'm not a huge ternary expression fan, but I can see they are being used in other places, so we can do
```suggestion
    tabGroupTitle () {
      return this.entity ? `Internal Services: ${entity.name}` : null
    },
```
or even if we are not strictly checking for nulls (I remember if you would like to render component in react you culdn't return any falsy value except nulls, so probably vuew is similar in that fashion)
```suggestion
    tabGroupTitle () {
      return this.entity && `Internal Services: ${entity.name}`
    },
```